### PR TITLE
Levono.SynapticsTouchpadDriver version 19.5.10.152

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111266&drop=dogfoodAlpha

### CODE REVIEW

**Comment by Trenly** [comment]
image

**Comment by upintheairsheep** [comment]
> image

Check for a flag to bypass it on a computer via calling the help flag, otherwise this is interactive only.

File: manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.installer.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

File: manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

File: manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Levono/SynapticsTouchpadDriver/19.5.10.152/Levono.SynapticsTouchpadDriver.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

File: manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.installer.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

File: manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

File: manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Levono.SynapticsTouchpadDriver
```
Comment (stephengillie) [inline manifests/l/Lenovo/SynapticsTouchpadDriver/19.5.10.152/Lenovo.SynapticsTouchpadDriver.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Lenovo.SynapticsTouchpadDriver
```

**Comment by stephengillie** [comment]
Had to swap the V and N in the ISV name.

**Pull request closed without merging**